src/server/routes.ts

```diff
@@ -190,6 +196,25 @@ router.post(
   },
 );
 
+//TODO finn bedre url
+router.post('/json2pdf', async (req: Request, res: Response) => {
```

Comment:
`manuelt-brev`
`frittstaende-brev`
`forenklet-brev`
...?

Reply:
manuelt-brev! Vi går for eden.

Reply:
Yes! Endret i 8f975bd